Library/Homebrew/formula_support.rb

```diff
@@ -28,13 +28,14 @@ def valid?
 
   def to_s
     return @explanation unless @explanation.empty?
+    os_name = OS::Mac.os_name(ENV["HOMEBREW_OSX_VERSION"])
```

Comment:
Yep, ideally we'd not have any user-facing messages that say "OS X" outside of `Library/Homebrew/os/mac/` or `Library/Homebrew/extend/os/mac.`